rbc/tests/test_omnisci_array_methods.py

```diff
@@ -19,13 +22,27 @@ def omnisci():
 ndarray_methods = [
     ('fill', (5, 4), [4.0, 4.0, 4.0, 4.0, 4.0]),
     ('max', (5, 4.0), 4.0),
-    ('max_empty', (0, ), -127),
+    ('max_empty_int8', (0, ), np.iinfo(np.int8).min + 1),
+    ('max_empty_int16', (0, ), np.iinfo(np.int16).min + 1),
+    ('max_empty_int32', (0, ), np.iinfo(np.int32).min + 1),
+    ('max_empty_int64', (0, ), np.iinfo(np.int64).min + 1),
+    ('max_empty_float32', (0, ), np.finfo(np.float32).min),
+    ('max_empty_float64', (0, ), np.finfo(np.float64).min),
     ('max_initial', (5, 4.0, 30.0), 30.0),
     ('mean', (5, 2.0), 2.0),
-    ('mean_empty_float', (0, ), np.nan),
-    ('mean_empty_int', (0, ), 0),
+    ('mean_empty_int8', (0, ), 0),
+    ('mean_empty_int16', (0, ), 0),
+    ('mean_empty_int32', (0, ), 0),
+    ('mean_empty_int64', (0, ), 0),
```

Comment:
`mean` always returns `float`, so, mean of empty integer sequence ought to be `nan` as well. As in numpy:
```python
>>> import numpy
>>> numpy.mean(numpy.array([1, 2], dtype=numpy.int64))
1.5
>>> numpy.mean(numpy.array([], dtype=numpy.int64))
RuntimeWarning: Mean of empty slice.
nan
```

```suggestion
   ('mean', (5, 2), 2.0),
   ('mean_empty_int8', (0, ), np.nan),
   ('mean_empty_int16', (0, ), np.nan),
   ('mean_empty_int32', (0, ), np.nan),
   ('mean_empty_int64', (0, ), np.nan),
```